tests/api/test_webhook.py

```diff
@@ -284,7 +284,7 @@ def test_webhook_update_by_staff(
     webhook_id = graphene.Node.to_global_id("Webhook", webhook.pk)
     variables = {
         "id": webhook_id,
-        "events": [WebhookEventTypeEnum.ORDER_CREATED.name],
+        "events": [WebhookEventTypeEnum.CUSTOMER_CREATED.name],
```

Comment:
In the mutation, you're using `set()` to create unique events. It could be tested that providing duplicated event actually doesn't break anything.